### PR TITLE
Add typing to simple-lru.

### PR DESCRIPTION
**Pull request by NN---**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [X] The package does not provide its own types, and you can not add them.
- [X] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [X] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [X] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/simple-lru/tslint.json [hunk 1]
```diff
@@ -0,0 +1,79 @@
+{
+    "extends": "dtslint/dt.json",
```
Comment (plantain-00) [inline types/simple-lru/tslint.json]: For new packages, the file should only `"extends": "dtslint/dt.json"`

**Review by plantain-00 — CHANGES_REQUESTED** [review]

File: types/simple-lru/tslint.json (hunk 1 above)
Comment (NN---) [reply types/simple-lru/tslint.json]: I used from other folders.
Np, I'll fix.

File: types/simple-lru/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
+// Type definitions for simple-lru
+// Project: https://github.com/smagch/simple-lru
+// Definitions by: NN <https://github.com/NN--->
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+declare module "simple-lru" {
```
Comment (RyanCavanaugh) [inline types/simple-lru/index.d.ts]: This shouldn't be written as a top-level declare. Please see other definitions for how to do this correctly.

**Review by RyanCavanaugh — CHANGES_REQUESTED** [review]

File: types/simple-lru/index.d.ts (hunk 2 above)
Comment (NN---) [reply types/simple-lru/index.d.ts]: I used same technique as other typings like https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/core-js/index.d.ts

```js
declare module "core-js" {
    export = core;
}
```

What is correct way to define ? Just remove the declare module and that's all ?
Thanks.

Comment (RyanCavanaugh) [reply types/simple-lru/index.d.ts]: Yeah, in this case you just have to remove the top and bottom lines of that snippet

**Review by plantain-00 — APPROVED** [review]

File: types/simple-lru/index.d.ts (hunk 2 above)
Comment (NN---) [reply types/simple-lru/index.d.ts]: Thanks. I will update the PR.

**Pull request merged**